src/controllers/handlers/http/get-community-invites-handler.ts

```diff
@@ -0,0 +1,63 @@
+import { InvalidRequestError } from '@dcl/platform-server-commons/dist/errors'
+import { EthAddress } from '@dcl/schemas'
+import { Community } from '../../../logic/community'
+import { HandlerContextWithPath, HTTPResponse } from '../../../types'
+import { errorMessageOrDefault } from '../../../utils/errors'
+
+/**
+ * This endpoint is used to retrieve potential invites for a user.
+ * It returns a list of communities where the authenticated user (inviter) is a member but the specified address (invitee) is not a member.
+ * This is used to render a list of potential communities to invite the user to.
+ */
+export async function getCommunityInvitesHandler(
+  context: Pick<
+    HandlerContextWithPath<'logs' | 'communities', '/v1/members/:address/invites'>,
+    'components' | 'params' | 'verification' | 'url'
+  >
+): Promise<HTTPResponse<Pick<Community, 'id' | 'name'>[]>> {
+  const {
+    components: { logs, communities },
+    params: { address: inviteeAddress },
+    verification
+  } = context
+
+  const logger = logs.getLogger('get-community-invites-handler')
+
+  try {
+    const inviterAddress = verification!.auth.toLowerCase()
+
+    if (inviterAddress === inviteeAddress) {
+      throw new InvalidRequestError('Users cannot invite themselves')
+    }
+
+    if (!EthAddress.validate(inviterAddress) || !EthAddress.validate(inviteeAddress)) {
```

Comment:
Just a question: isn't `inviterAddress` validated with the signedFetchMiddleware?

Reply:
I checked and it seems it isn't validating addresses explicitly with this function!